memstore/schema.go

```diff
@@ -151,7 +151,7 @@ func (t *TableSchema) SetDefaultValue(columnID int) {
 		if !ok {
 			// Should no happen since the enum value should already be created.
 			utils.GetLogger().With(
-				"data_type", t.Schema.Columns[columnID].Type,
+				"data_type", memCom.DataTypeName[dataType],
```

Comment:
can we cache the dataTypeName since it's used multiple times

Reply:
Done